src/vstest.console/Program.cs

```diff
@@ -20,44 +17,10 @@ public static class Program
         /// <param name="args">Arguments provided on the command line.</param>
         /// <returns>0 if everything was successful and 1 otherwise.</returns>
         public static int Main(string[] args)
-        {            
-            var debugEnabled = Environment.GetEnvironmentVariable("VSTEST_RUNNER_DEBUG");
-            if (!string.IsNullOrEmpty(debugEnabled) && debugEnabled.Equals("1", StringComparison.Ordinal))
-            {
-                ConsoleOutput.Instance.WriteLine("Waiting for debugger attach...", OutputLevel.Information);
-
-                var currentProcess = System.Diagnostics.Process.GetCurrentProcess();
-                ConsoleOutput.Instance.WriteLine(
-                    string.Format("Process Id: {0}, Name: {1}", currentProcess.Id, currentProcess.ProcessName),
-                    OutputLevel.Information);
-
-                while (!Debugger.IsAttached)
-                {
-                    Thread.Sleep(1000);
-                }
-
-                Debugger.Break();
-            }
-
-            SetCultureSpecifiedByUser();
-
-            return new Executor(ConsoleOutput.Instance).Execute(args);
-        }
-
-        private static void SetCultureSpecifiedByUser()
         {
-            var userCultureSpecified = Environment.GetEnvironmentVariable(CoreUtilities.Constants.DotNetUserSpecifiedCulture);
-            if(!string.IsNullOrWhiteSpace(userCultureSpecified))
-            {
-                try
-                {
-                    CultureInfo.DefaultThreadCurrentUICulture = CultureInfo.CreateSpecificCulture(userCultureSpecified);
-                }
-                catch(Exception)
-                {
-                    ConsoleOutput.Instance.WriteLine(string.Format("Invalid Culture Info: {0}", userCultureSpecified), OutputLevel.Information);
-                }
-            }
+            DebuggerBreakpoint.WaitForDebugger("VSTEST_RUNNER_DEBUG");
```

Comment:
So... Should we move?

Reply:
What do you mean?

Reply:
> Some comment about possibly moving the names for these variables to sharedproj too..

:) Just marking to spot so we don't miss it if we decide move it.

Reply:
I am for not moving.